src/sentry/static/sentry/app/components/dropdownAutoCompleteMultiRow.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import CheckboxFancy from 'app/components/checkboxFancy';
+import space from 'app/styles/space';
+
+class DropdownAutoCompleteMultiRow extends React.Component {
```

Comment:
Is this always a multi row? Since there is a prop `noMultiFeature` I assume not and we should drop that part from the name?

Reply:
yeah good call. Maybe instead it should be `globalSelectionHeaderRow`? This row isn't used with every dropdownAutocomplete instance—only with `MultipleEnvironmentSelector` and `MultipleProjectSelector`.